logging/Limiting_Access_to_Logs.md

```diff
@@ -168,14 +177,19 @@ user.sh script.  For example, to create a user called "chen" and link the user t
 access controls, submit the following command:
 
 <pre>
-/logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
---namespace _all_ --tenant _all_
+logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
+--namespace <i>namespace</i> --tenant <i>tenant</i>
 </pre>
 
 Where: 
 
 - *user_name* is "chen".
 - *password* is the password to sign in to Kibana as the user "chen".
+- *namespace* is "mynamespace".
+- *tenant* is "mytenant1".
```

Comment:
The WHERE info doesn't match the code block.  The code block shows the calling syntax, the description of the various fields appears to reflect sample values from a sample call that is no longer present.

Reply:
We can either present it as a pure example (in which case the values are used in the code block) or as a syntax with examples of values shown below in the where section. 
It sounds like your preference is to make it a pure example.

Reply:
Here is the revision (FYI, I have removed the note about using `_all_` because we have the example further down):

You can create new users and link them to these access controls by running the 
user.sh script.  For example, to create a user called "chen" and link the user to these 
access controls, submit the following command:

<pre>
logging/bin/user.sh CREATE --user chen --password pw2022zxc 
--namespace mynamespace --tenant mytenant1
</pre>